### PR TITLE
Textintro component

### PR DESCRIPTION
**Pull request by CSCairney-GL**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -19,3 +19,7 @@ body {
   margin: 0;
   padding: 0;
 }
+
+* {
+  font-family: 'HelveticaNeue-Light', 'Helvetica Neue Light', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif;
+}
```
Comment (CSCairney-GL) [inline src/App.css:25]: This was removed from the Homepage.css that got deleted and put into the app.css with the other global css changes like body and root colours.

File: src/pages/styles/HomePage.css [hunk 2]
```diff
@@ -1,74 +0,0 @@
...
-}
-
-@media screen and (max-width: 800px) {
-  .homepage-title {
-    font-size: 40px;
-  }
-  .homepage-subheading {
-    font-size: 20px;
-  }
-}
```
Comment (CSCairney-GL) [inline src/pages/styles/HomePage.css:74]: Removed as the remaining CSS was just for the hard coded TextIntro that has now be moved to TextIntro.css.

**Review by MihneaDinuGL — APPROVED** [review]
Nice work! I was wondering if you'd like to implement the Markdown component into the actual text? I'm not sure it's necessary but it might be useful, I'd be happy to jump on a call and do it or we can do it tomorrow morning if you're interested :)

**Comment by CSCairney-GL** [comment]
> Nice work! I was wondering if you'd like to implement the Markdown component into the actual text? I'm not sure it's necessary but it might be useful, I'd be happy to jump on a call and do it or we can do it tomorrow morning if you're interested :)

Thanks Alex and yeah that would be great i'll message you on teams to arrange a time now :)

**Review by jab-khanGL — APPROVED** [review]

**Review by sourcesmith — APPROVED** [review]

**Pull request merged**